plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -184,6 +192,18 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
         makeBuiltinMeaning
             BS.drop
             (runCostingFunTwoArguments . paramDropByteString)
+    toBuiltinMeaning SliceByteString =
+        makeBuiltinMeaning
+            (\from to xs -> BS.take (to - from + 1) (BS.drop from xs))
```

Comment:
huh. Do we not have a more efficient implementation of this? I guess this is okay.

Reply:
Well, this implementation was the most obvious way to get it work. I can try to implement it via [`ForeignPtr`](https://hackage.haskell.org/package/bytestring-0.11.1.0/docs/Data-ByteString-Internal.html#v:toForeignPtr).

Reply:
Nah, it seems okay. It'll probably only be a multiple slower rather than algorithmically slower, so that seems okay.